### PR TITLE
Support for tokenize Japanese document with emojis

### PR DESCRIPTION
**Pull request by uekann**

### CODE REVIEW

File: brack-tokenizer/src/utils.rs [hunk 1]
```diff
@@ -3,14 +3,27 @@ use crate::{
...
+use unicode_segmentation::UnicodeSegmentation;
+
+pub fn separate(s: &str) -> (String, String) {
+    // if s == "" {
+    //     return ('\0'.to_string(), String::new());
+    // }
+    // let graphemes = s.graphemes(true);
+    // if graphemes.count() == 1 {
+    //     return (s.graphemes(true).nth(0).unwrap().to_string(), String::new());
+    // }
```
Comment (momeemt) [inline brack-tokenizer/src/utils.rs:15]: Would you remove this comments?

**Review by momeemt — CHANGES_REQUESTED** [review]

**Comment by uekann** [comment]
Sorry, I forgot to erase these comments.
I fixed it!

**Comment by momeemt** [comment]
@uekann 
Thank you for your handling!
You should keep pull requests as simple as possible, because reviewers can easily see through all changes, and we can quickly revert a change if we discover a serious bug. So I want you to split into more than two pull requests if there are multiple changes. You can execute git revert or git cherry-pick to move another change to a new branch.
I'll merge what fixes the Unicode character handling, but adding the debug configuration should be reviewed another time.

**Review by momeemt — APPROVED** [review]
LGTM!

**Pull request merged**